webassembly.html

```diff
@@ -317,6 +317,13 @@ <h3 id="w3c-coordination">W3C Groups</h3>
             process.</dd>
           </dl>
           <dl>
+            <dt><a href="https://www.w3.org/2008/webapps/">Web Applications Working Group</a></dt>
```

Comment:
There is no Web Applications WG any longer. It was replaced by the Web Platform WG